calculate-returns.py

```diff
@@ -0,0 +1,61 @@
+# Python script to calculate the pension fund for mutual funds
+import pandas as pd
+
+# Import the dataframe from excel file
+df_pension = pd.read_excel('./data/16-Apr-2021.xls', header=None)
+
+# Calculate the rows in third column where the value is NaN
+missing_list = list(df_pension.index[df_pension[df_pension.columns[3]].isna()])
+
+# Split dataframes using the missing_list
```

Comment:
Add these comments as doc-strings, instead of as comments. Doc-strings have the nicety of being shown by `help` and other such Python tools.